docs/data/joy/getting-started/templates/sign-in-side/App.tsx

```diff
@@ -154,58 +153,83 @@ export default function JoySignInSideTemplate() {
               },
             }}
           >
-            <div>
-              <Typography component="h1" fontSize="xl2" fontWeight="lg">
-                Sign in to your account
-              </Typography>
-              <Typography level="body-sm" sx={{ my: 1, mb: 3 }}>
-                Welcome back
-              </Typography>
-            </div>
-            <form
-              onSubmit={(event: React.FormEvent<SignInFormElement>) => {
-                event.preventDefault();
-                const formElements = event.currentTarget.elements;
-                const data = {
-                  email: formElements.email.value,
-                  password: formElements.password.value,
-                  persistent: formElements.persistent.checked,
-                };
-                alert(JSON.stringify(data, null, 2));
-              }}
-            >
-              <FormControl required>
-                <FormLabel>Email</FormLabel>
-                <Input type="email" name="email" />
-              </FormControl>
-              <FormControl required>
-                <FormLabel>Password</FormLabel>
-                <Input type="password" name="password" />
-              </FormControl>
-              <Box
-                sx={{
-                  display: 'flex',
-                  justifyContent: 'space-between',
-                  alignItems: 'center',
-                }}
+            <Stack gap={4} sx={{ mb: 2 }}>
+              <Stack gap={1}>
+                <Typography level="h3">Sign in</Typography>
+                <Typography level="body-sm">
+                  New to company?
```

Comment:
```suggestion
                  New to company?{" "}
```

We're missing a space between "New to company" and "Sign up".

Additionally, for all the logos, I'd probably use a fake company name instead of nothing ⎯ folks usually use "Acme" as a fake one.